docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: "3.5"
```

Comment:
This is obsolete and led to a warning on startup.
See https://docs.docker.com/reference/compose-file/version-and-name/.